clang/test/CodeGen/aarch64-args.cpp

```diff
@@ -65,3 +65,9 @@ EXTERNC struct SortOfEmpty sort_of_empty_ret(void) {
   struct SortOfEmpty e;
   return e;
 }
+
+// CHECK-GNU-CXX: define{{.*}} i32 @empty_align_arg(i128 %a.coerce, i32 noundef %b)
+struct EmptyAlign { long long int __attribute__((aligned)) : 0; };
```

Comment:
You should also have test coverage for an empty aligned structure (not just an alignment on a zero-width bit-field, which I'm still not certain what that actually means or why we shouldn't reject that code).